### PR TITLE
LPS-96723 Remove pointless comma operator

### PR DESCRIPTION
**Pull request by wincent**
This pattern:

    fn(...(null, args));

looks like a mistranslation of:

    fn.apply(null, args);

ie. call function `fn` with its context (`this`) set to `null`, and passing it `args`.

However, [the comma operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comma_Operator) has no effect. It evaluates the first operand (`null`), discards it, then evaluates the second one (`args`) and uses that as the result of the expression:

In other words, it can be rewritten as just:

    fn(...args);

### CODE REVIEW

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/65631fee90...74bc4d80a9

**Comment by brianchandotcom** [comment]
@wincent I backported this to 7.2.x because this ticket was already flagged as being in 7.2.x

In the future, please use new LPS ticket so it's easier for support to track (and I won't backport it)

**Comment by wincent** [comment]
Got it, @brianchandotcom. Thanks.
